src/app/(routes)/customer/roadmap/[...slug]/page.tsx

```
@@ -175,225 +180,62 @@ const Page = () => {
     };
 
     const handleNodeClick = (event: React.MouseEvent, node: Node<Partial<CustomNodeData>>) => {
-        console.log('Clicked Node ID:', node.id); // Debug the node ID
-        setSelectedNode(node); // Ensure state is updated
+        console.log('Clicked Node ID:', node.id); // Debug log
+        if (node && node.id) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)